src/app/app.module.ts

```diff
@@ -29,5 +35,11 @@ import { PagesNotFoundPageComponent } from './pages/not-found/pages-not-found-pa
     CommonComponentsModule,
   ],
   bootstrap: [ AppComponent ],
+  providers: [ {
```

Comment:
Ayyy, good news / bad news - Lets remove this global service and load data normally on the categories page. When reviewing this I've notice so many issues with this approach that it will be much better do it normally.
For example now we are not handling errors at all - and because we should anyway re-fetch this data when going to categories page or transaction editor -> this means if there was error we will first see error, then it will re-fetch data and then we will still see error OR this time re-fetching will work and we will see data after first seeing error - so it will behave super weird.
So please remove this approach and do it in the same way we load transactions now.